### ISSUE TITLE
Starting error

### ISSUE THREAD

**Issue by stuckmcx** [knxd/knxd#249]
Hello,

after upgrading from v0.12 to v0.14 i have a Problem when knxd start.

```
Apr 20 21:12:56 knxd systemd[1]: Starting KNX Daemon...
Apr 20 21:12:57 knxd knxd[1210]: Layer 3 [13:server/Server     0.003] registerLink: 13:server
Apr 20 21:12:57 knxd knxd[1210]: Layer 3 [13:server/Server     0.036] Start: cfg:server
Apr 20 21:12:57 knxd knxd[1210]: Layer 5 [13:server/Server     0.042] down => >up
Apr 20 21:12:57 knxd knxd[1210]: Layer 8 [13:server/Server     0.042] Open
Apr 20 21:12:57 knxd knxd[1210]: Layer 0 [13:server/Server     0.043] Open
Apr 20 21:12:57 knxd knxd[1210]: Layer 0 [13:server/Server     0.043] Opened
Apr 20 21:12:57 knxd knxd[1210]: Layer 8 [24:router/driver     0.043] OpenD
Apr 20 21:12:57 knxd knxd[1210]: Layer 8 [24:router/driver     0.045] OpenedD
Apr 20 21:12:57 knxd knxd[1210]: N00000000: [25:router.pace_] The 'pace' filter without a queue acts globally.
Apr 20 21:12:57 knxd knxd[1210]: Layer 3 [23:router/ConnC      0.045] registerLink: 23:router_23
Apr 20 21:12:57 knxd knxd[1210]: Layer 3 [23:router/ConnC      0.045] Start: cfg:router
Apr 20 21:12:57 knxd knxd[1210]: Layer 5 [23:router/ConnC      0.045] down => >up
Apr 20 21:12:57 knxd knxd[1210]: Layer 5 [23:router/ConnC      0.045] Starting
Apr 20 21:12:57 knxd knxd[1210]: Layer 5 [23:router/ConnC      0.046] >up => up
Apr 20 21:12:57 knxd knxd[1210]: Layer 4 [23:router/ConnC      0.046] up
Apr 20 21:12:57 knxd knxd[1210]: Layer 5 [23:router/ConnC      0.046] Started
Apr 20 21:12:57 knxd knxd[1210]: Layer 4 [23:router/ConnC      0.046] up
Apr 20 21:12:57 knxd knxd[1210]: Layer 8 [13:server/Server     0.046] Opened
Apr 20 21:12:57 knxd knxd[1210]: Layer 5 [13:server/Server     0.046] >up => up
Apr 20 21:12:57 knxd knxd[1210]: Layer 4 [13:server/Server     0.046] up
Apr 20 21:12:57 knxd knxd[1210]: Layer 5 [13:server/Server     0.046] Started
Apr 20 21:12:57 knxd knxd[1210]: Layer 4 [13:server/Server     0.046] up
Apr 20 21:12:57 knxd systemd[1]: knxd.service: main process exited, code=killed, status=6/ABRT
Apr 20 21:12:57 knxd systemd[1]: Failed to start KNX Daemon.
Apr 20 21:12:57 knxd systemd[1]: Unit knxd.service entered failed state.
Apr 20 21:12:57 knxd knxd[1210]: knxd: ../../src/libserver/lowlevel.h:94: virtual void LowLevelIface::do_send_Local(CArray&, int): Assertion `!raw' failed.
```
My Startup Options are:

`KNXD_OPTS="-e 1.1.99 -E 1.1.99:8 -u /tmp/knx -t 0xfff -f 9 -D -T -R -S -B single -b usb:"`

Another problem is that:

```
root@knxd:~# knxd --help
Usage: knxd configfile [main-section]
```

Thanks for your help.

Regards Marc

**Comment by stuckmcx** [comment]
Hello,

Same Problem with v0.14.12

```
Apr 25 21:32:39 knxd systemd[1]: knxd.service holdoff time over, scheduling restart.
Apr 25 21:32:39 knxd systemd[1]: Stopping KNX Daemon...
Apr 25 21:32:39 knxd systemd[1]: Starting KNX Daemon...
Apr 25 21:32:40 knxd knxd[19594]: Layer 3 [10:server/Server     0.002] registerLink: 10:server
Apr 25 21:32:40 knxd knxd[19594]: Layer 3 [10:server/Server     0.037] Start: cfg:server
Apr 25 21:32:40 knxd knxd[19594]: Layer 5 [10:server/Server     0.042] down => >up
Apr 25 21:32:40 knxd knxd[19594]: Layer 8 [10:server/Server     0.042] Open
Apr 25 21:32:40 knxd knxd[19594]: Layer 0 [10:server/Server     0.043] Open
Apr 25 21:32:40 knxd knxd[19594]: Layer 0 [10:server/Server     0.043] Opened
Apr 25 21:32:40 knxd knxd[19594]: Layer 8 [21:router/driver     0.044] OpenD
Apr 25 21:32:40 knxd knxd[19594]: Layer 8 [21:router/driver     0.045] OpenedD
Apr 25 21:32:40 knxd knxd[19594]: N00000000: [22:router.pace_] The 'pace' filter without a queue acts globally.
Apr 25 21:32:40 knxd knxd[19594]: Layer 3 [20:router/ConnC      0.046] registerLink: 20:router_20
Apr 25 21:32:40 knxd knxd[19594]: Layer 3 [20:router/ConnC      0.046] Start: cfg:router
Apr 25 21:32:40 knxd knxd[19594]: Layer 5 [20:router/ConnC      0.046] down => >up
Apr 25 21:32:40 knxd knxd[19594]: Layer 5 [20:router/ConnC      0.046] Starting
Apr 25 21:32:40 knxd knxd[19594]: Layer 5 [20:router/ConnC      0.046] >up => up
Apr 25 21:32:40 knxd knxd[19594]: Layer 4 [20:router/ConnC      0.046] up
Apr 25 21:32:40 knxd knxd[19594]: Layer 5 [20:router/ConnC      0.046] Started
Apr 25 21:32:40 knxd knxd[19594]: Layer 4 [20:router/ConnC      0.046] up
Apr 25 21:32:40 knxd knxd[19594]: Layer 8 [10:server/Server     0.046] Opened
Apr 25 21:32:40 knxd knxd[19594]: Layer 5 [10:server/Server     0.046] >up => up
Apr 25 21:32:40 knxd knxd[19594]: Layer 4 [10:server/Server     0.046] up
Apr 25 21:32:40 knxd knxd[19594]: Layer 5 [10:server/Server     0.046] Started
Apr 25 21:32:40 knxd knxd[19594]: Layer 4 [10:server/Server     0.046] up
Apr 25 21:32:40 knxd systemd[1]: knxd.service: main process exited, code=killed, status=6/ABRT
Apr 25 21:32:40 knxd systemd[1]: Failed to start KNX Daemon.
Apr 25 21:32:40 knxd systemd[1]: Unit knxd.service entered failed state.
Apr 25 21:32:40 knxd knxd[19594]: knxd: ../../src/libserver/lowlevel.h:94: virtual void LowLevelIface::do_send_Local(CArray&, int): Assertion `!raw' failed.
```
`
KNXD_OPTS="-e 1.1.99 -E 1.1.99:8 -t 0xfff -f 9 -D -T -R -S -B single -b usb:"
`
Any Ideas?

Thanks
Marc

**Comment by smurfix** [comment]
Reproduced. Investigating.

**Comment by smurfix** [comment]
Should be fixed in the current version (v0.14.13)

**Comment by stuckmcx** [comment]
Hey!

Thanks this is Fixed. But now a have an other Problem. Open new Issue.

Thanks
